rust/processor/src/models/token_v2_models/v2_token_datas.rs

```diff
@@ -99,21 +77,13 @@ impl TokenDataV2 {
         if let Some(inner) = &TokenV2::from_write_resource(write_resource, txn_version)? {
             let token_data_id = standardize_address(&write_resource.address.to_string());
             let mut token_name = inner.get_name_trunc();
-            // Get maximum, supply, and is fungible from fungible asset if this is a fungible token
-            let (mut maximum, mut supply, mut decimals, mut is_fungible_v2) =
-                (None, BigDecimal::zero(), 0, Some(false));
+            let mut is_fungible_v2 = Some(false);
```

Comment:
why not always null?

Reply:
this will be the only table that indexes the fungibility of the token. 
for v1 tokens, is_fungible_v2 is null. for v2 tokens, it can be true or false.

Reply:
yea but isn't it always false after? b/c we're no longer indexing fungible tokens here? Or are we?

Reply:
we are. is_fungible_v2 = true if there's a metadata resource at the token address.